### PR TITLE
Instruction on how to operate

### PR DESCRIPTION
**Pull request by Thomas-dot-G**
Explanation of how to create your own svg file

### CODE REVIEW

**Comment by anqitong** [comment]
...svg.writeSVGFile();

When this method is called, it checks if there are other svg files with the same name. If yes, the svg file cannot be created and a warning is displayed. If no, then the file is successfully created.

(can you add this?)

File: README.md [hunk 1]
```diff
@@ -4,7 +4,50 @@ This project aims to create SVG images through the command of a user.
 
 The subject of the subject is described [here](http://www.grall.name/teaching/softwareArchitecture/2016/project_vectorGraphics.html) (in French)
 
-Team Members:
+## How to use the software:
+
+Once the project has been retrieve, you can use it to create SVG files.
```
Comment (anqitong) [inline README.md]: Once the project is retrieved

File: README.md [hunk 2]
```diff
@@ -4,7 +4,50 @@ This project aims to create SVG images through the command of a user.
...
+
+You can reapeat the Step 2 as many times as you want.
+
+### Step 3:
+Gather your shapes to add them to a drawing:
+
+		ArrayList<Shape> shapes = new ArrayList<Shape>();
+		shapes.add(c);
+
+Once it has been done, you'll need to create a SVGFile, which is the presentation that will create a SVG File :
```
Comment (anqitong) [inline README.md]: Once it is done

File: README.md [hunk 3]
```diff
@@ -4,7 +4,50 @@ This project aims to create SVG images through the command of a user.
...
+
+		SVGFile svg = new SVGFile("name_of_the_file");
```
Comment (starsasumi) [inline README.md]: Can you add comments like "name without .svg"? I used to add ".svg" in here and get something like xyz.svg.svg

File: README.md [hunk 4]
```diff
@@ -4,7 +4,50 @@ This project aims to create SVG images through the command of a user.
 
 The subject of the subject is described [here](http://www.grall.name/teaching/softwareArchitecture/2016/project_vectorGraphics.html) (in French)
 
-Team Members:
+## How to use the software:
+
+Once the project has been retrieved, you can use it to create SVG files.
```
Comment (anqitong) [inline README.md]: Once the project is retrieved

**Comment by anqitong** [comment]
Fine for me now.

**Comment by Thomas-dot-G** [comment]
issue #31

**Comment by starsasumi** [comment]
Ok for me.

**Pull request merged**